src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/RazorSemanticTokensInfoService.cs

```diff
@@ -70,6 +70,10 @@ internal class RazorSemanticTokensInfoService : IRazorSemanticTokensInfoService
         {
             csharpSemanticRanges = await GetCSharpSemanticRangesAsync(codeDocument, textDocumentIdentifier, range, razorSemanticTokensLegend, documentContext.Version, correlationId, cancellationToken).ConfigureAwait(false);
         }
+        catch (OperationCanceledException)
```

Comment:
Noticed in the logs we were reporting this as an issue, when its totally normal, and in fact it should probably happen a little more than it does now!